File: two_sum.java
public class two_sum 
{
    public static int[] twoSum(int[] nums, int target) 
    {
        for (int n = 0; n < nums.length; n++)
        {
            for (int i = n + 1; i < nums.length; i++)
            {
                if (nums[n] + nums[i] == target)
                {
                    int [] result = {i,n};
                    return result;
                }
            }
        }
        return null;
    }

    public static void main(String[] args)
    {
        int [] nums = {3,2,4};
        System.out.println(twoSum(nums, 6));
    }
}

File: two_sum.py
def twoSum(nums, target):
    hash = {}
    for i in range(len(nums)):
        y = target - nums[i]
        if (y in hash):
            return ([i, hash[y]])
        else:
            hash[nums[i]] = i
        
def main():
    nums = [2,7,11,15]
    target = 9
    print(twoSum(nums, target))

if __name__ == "__main__":
    main()